source/src/protocols/cyclic_peptide_predict/SimpleCycpepPredictApplication.cc

```diff
@@ -803,6 +813,25 @@ SimpleCycpepPredictApplication::initialize_from_options(
 		}
 	}
 
+	//CWT store lan and melan positions
+	if ( option[basic::options::OptionKeys::cyclic_peptide::lanthionine_positions].user() ) {
+		core::Size const nlanres(option[basic::options::OptionKeys::cyclic_peptide::lanthionine_positions]().size());
+		runtime_assert_string_msg( nlanres > 0, "Error in simple_cycpep_predict application: The \"-cyclic_peptide:lanthionine_positions\" commandline option must be followed by a list of residues to form lanthionine rings." );
+		//runtime_assert_string_msg( nlanres % 2 == 0, "Error in simple_cycpep_predict application: The \"-cyclic_peptide:lanthionine_positions\" commandline option must be followed by a list of residues, where the number of residues in the list is a multiple of two.  Pairs of residues must form lanthionine rings." );
+		//Only allowing two specified residues at the moment
+		runtime_assert_string_msg( nlanres == 2, "Error in simple_cycpep_predict application: The \"-cyclic_peptide:lanthionine_positions\" commandline option must be followed by a list of two residues. Pairs of residues must form lanthionine rings." );
```

Comment:
This seems a bit off. Your loop seems to indicate that you should be able to handle multiple lantionine bonds. Is there a particular reason you're limiting it to only a single bond (two residues) at the moment? You may want to clarify the reason for that here, so the next person has a better sense of what would need to be addressed.

Reply:
The reason is that SimpleCycpepPredict is not currently able to adjacent rings or overlapping rings. I added a comment in the code to address this.